.env.default

```diff
@@ -17,10 +17,10 @@ DEV_MODE=true
 LAGOON_ENVIRONMENT_TYPE=local
 
 # https://govcms.gov.au/wiki_vars#LOCALDEV_URL
-LOCALDEV_URL=http://govcms8-paas.docker.amazee.io
+LOCALDEV_URL=http://govcms8-scaffold-satis.docker.amazee.io
```

Comment:
is this intentional change? It used to be a generic website URL

Reply:
This was accidental, being fixed. There may be changes to this but not in this PR.